### DISCUSSION TITLE
Country name error

### DISCUSSION THREAD

**Discussion by YiJhu** [timschneeb/GalaxyBudsClient#269] [Releases]
In fact, Taiwan belongs to the "**Republic of China**"; 
it does not belong to the "Province of China".
Hope you can correct [THIS](https://github.com/ThePBone/GalaxyBudsClient/blame/master/meta/tw.md) MARKDOWN file.

**Comment by timschneeb** [comment]
Hey, I'm using an external library to resolve these country names based on the country codes.
In the best case, I'll try to find a library that is able to convert country codes to the actual language name (not country name).

**Reply by YiJhu** [reply]
Ok, have a nice day
